test/test_torch.py

```diff
@@ -18872,51 +18884,57 @@ def test_split_view(self, device):
 # the results. In-place variants of the ops can also be run.
 
 # Lists of dtypes to instantiate tensor op test variants.
-_types = [
+_types = (
     torch.half, torch.float, torch.double,
     torch.int8, torch.short, torch.int, torch.long,
     torch.uint8
-]
+)
 
-_types_no_half = [
+_types_no_half = (
     torch.float, torch.double,
     torch.int8, torch.short, torch.int, torch.long,
     torch.uint8
-]
+)
 
 # _types2 adds bfloat16 type to  _types only on ROCm. Should eventually be unified
 # with _types when bfloat16 bringup is complete on all platforms.
-_types2 = _types + [torch.bfloat16] if TEST_WITH_ROCM else _types
+_types2 = _types + (torch.bfloat16,) if TEST_WITH_ROCM else _types
+
+_float_types = (torch.half, torch.float, torch.double)
+
+_tfloat_types = (tfloat32,)
```

Comment:
Per the above comment on new dtypes, do you really need new dtypes or is the decorator sufficient?

Reply:
fixed